man/systemd-repart.xml

```diff
@@ -364,6 +364,16 @@
         <xi:include href="version-info.xml" xpointer="v252"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--signing-engine=</option></term>
```

Comment:
hmm, "engine" is the old concept that is deprecated, no? hence please don't add new options named in a way that already have built-in the idea that they are dead. Hence, maybe call this `--signing-provider=` instead?

(i am fine if the actual implementation actually supports the old stuff too if we can, but the exposed interface should always be forward-looking, not backwards-looking)